### PR TITLE
Update README so it correctly reflects project structure, current release number and webserver, RPC ports.

### PR DESCRIPTION
**Pull request by kasiastreich**
Also changed runExampleClientRPC task so RPC port matches what is in configs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,7 +301,7 @@ To run the client:
 **Via IntelliJ:**
 
 Select the 'Run Example RPC Client' run configuration which, by default,
-connects to NodeA (Artemis port 10004). Click the Green Arrow to run the
+connects to NodeA (Artemis port 10006). Click the Green Arrow to run the
```
Comment (patrickkuo) [inline README.md]: RPC port

File: README.md [hunk 2]
```diff
@@ -80,9 +80,9 @@ Change directories to the newly cloned repo:
 
      cd cordapp-tutorial
      
-Check out the latest milestone release (currently `release-M8.2`):
+Check out the latest milestone release (currently `release-M9.2`):
```
Comment (rnicoll) [inline README.md]: Wonder if we want to specify a release at all, or point people at https://github.com/corda/corda/tags ?

Comment (kasiastreich) [inline README.md]: Yes, good idea, otherwise it becomes a nightmare.

Comment (Clintonio) [inline README.md]: Keep the release number. Developers copy and paste and for developer experience purposes _we_ should maintain this tiny bit of documentation to make it easier for people to get started.

**Pull request merged**